File: xenon-android-library/src/main/kotlin/com/abubusoft/xenon/mesh/tiledmaps/TileAnimationFrame.java
package com.abubusoft.xenon.mesh.tiledmaps;

/**
 * Layer che fa parte di una animazione.
 * 
 * @author Francesco Benincasa
 * 
 */
public class TileAnimationFrame {

	public TileAnimationFrame(Layer layerValue, long durationValue) {
		layer = layerValue;
		duration = durationValue;
	}

	/**
	 * durata del frame in millisecondi
	 */
	public long duration;

	/**
	 * frame associato
	 */
	public final Layer layer;

}

File: xenon-android-library/src/main/kotlin/com/abubusoft/xenon/mesh/tiledmaps/TileAnimationFrame.kt
package com.abubusoft.xenon.mesh.tiledmaps

/**
 * Layer che fa parte di una animazione.
 *
 * @author Francesco Benincasa
 */
class TileAnimationFrame(
    /**
     * frame associato
     */
    val layer: Layer,
    /**
     * durata del frame in millisecondi
     */
    var duration: Long
)